frontend/src/components/Header.tsx

```diff
@@ -0,0 +1,88 @@
+import HeaderDesktop from './HeaderDesktop';
+import HeaderMobile from './HeaderMobile';
+import { useEffect, useState} from 'react';
+import { FaMoon, FaSun } from "react-icons/fa";
+import LinksMenu from './LinksMenu';
+
+interface theme {
+    setDarkTheme: void;
```

Comment:
Alteração: tipar a função corretamente:

```suggestion
    setDarkTheme: () => void;
```